dll/win32/shell32/utils.h

```diff
@@ -90,3 +90,39 @@ SHELL_CreateFallbackExtractIconForNoAssocFile(REFIID riid, LPVOID *ppvOut)
     const int id = IDI_SHELL_DOCUMENT;
     return SHELL_CreateShell32DefaultExtractIcon(id > 1 ? -id : 0, riid, ppvOut);
 }
+
+struct ClipboardViewerChain
+{
+    HWND m_hWndNext;
+
+    ClipboardViewerChain() : m_hWndNext(HWND_BOTTOM) {}
```

Comment:
nit: You could use the in-class initializer and completely drop the constructor.